src/components/Dashboard/Alerts/InDissolution.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <v-expansion-panel id="in-dissolution-panel">
+    <v-expansion-panel-header
+      hide-actions
+      class="d-flex justify-space-between px-6 py-5"
+    >
+      <h3>
+        <v-icon
+          left
+          color="error"
+        >
+          mdi-alert
+        </v-icon>
+        <span>Urgent - this business is in the process of being dissolved</span>
+      </h3>
+      <v-btn
+        text
+        color="primary"
+        class="details-btn my-n1"
+        @click.stop="togglePanel()"
+      >
+        <span color="primary">{{ showPanel ? "Hide Details" : "View Details" }}</span>
+        <v-icon
+          right
+          color="primary"
+        >
+          {{ showPanel ? "mdi-chevron-up" : "mdi-chevron-down" }}
+        </v-icon>
+      </v-btn>
+    </v-expansion-panel-header>
+
+    <v-expansion-panel-content eager>
+      <p class="mb-0">
+        This means that the business will be struck from the Corporate Registry in
+        <strong>{{ daysLeft }} days</strong> due to overdue annual reports.
+        Please file the annual reports immediately to bring this business back into good standing
+        or request a delay of dissolution if more time is needed.
+      </p>
+      <p class="mb-0 pt-5">
+        For assistance, please contact BC Registries staff:
+      </p>
+      <ContactInfo class="pt-5" />
+    </v-expansion-panel-content>
+  </v-expansion-panel>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+import { BusinessWarningIF } from '@/interfaces'
+import { ContactInfo } from '@/components/common'
+import { Getter } from 'pinia-class'
+import { useBusinessStore, useRootStore } from '@/stores'
+import { DateUtilities } from '@/services'
+import { WarningTypes } from '@/enums'
+
+@Component({
+  components: { ContactInfo }
+})
+export default class InDissolution extends Vue {
+  @Prop({ required: true }) readonly showPanel!: boolean
+
+  @Getter(useBusinessStore) getBusinessWarnings!: BusinessWarningIF[]
+  @Getter(useRootStore) getCurrentDate!: string
+
+  /** Return the number of days left for the business until it is dissolved. */
+  get daysLeft (): number {
+    const warning = this.getBusinessWarnings.find(item =>
+      item.warningType?.includes(WarningTypes.INVOLUNTARY_DISSOLUTION)
+    )
+    const targetDissolutionDate = warning?.data?.targetDissolutionDate
+
+    return DateUtilities.daysBetweenTwoDates(new Date(this.getCurrentDate), new Date(targetDissolutionDate)) || -1
```

Comment:
The -1 here is just a fallback. Theoretically, we should never get that.

Reply:
So if the days between the two dates is 0 (ie, false) then you use -1 instead? I think that would look weird.

Perhaps you want to do something specific if the result is NaN (ie, display Unknown)?

Reply:
Good point. I was actually thinking the fallback would happen if we were unable to grab both dates for whatever reason.

I'm thinking, I'll remove that fallback altogether. The `daysBetweenTwoDates` function already has safety checks for all the cases.

Reply:
This conversation is continued below. Resolving this one.